ledger/psbt.ts

```diff
@@ -153,7 +155,16 @@ export async function signLedgerPSBT({
     });
   }
 
-  if (await embellishNativeSegwitInputs(txn, nativeSegwitPubKey, network, addressIndex, masterFingerPrint)) {
+  if (
+    await embellishNativeSegwitInputs(
+      txn,
+      nativeSegwitPubKey,
+      esploraProvider,
+      network,
+      addressIndex,
+      masterFingerPrint,
+    )
```

Comment:
There are a couple of functions like this that we should be moving to an object arg instead. I'd like to tackle that as a separate PR once txn consolidation is in and in use as a lot of the affected functions will be removed.